### PR TITLE
Add ledgerdb cli and open ledger

### PR DESCRIPTION
**Pull request by wjuan-mob**

### CODE REVIEW

**Comment by wjuan-mob** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2** <a href="https://app.graphite.dev/github/pr/mobilecoinofficial/deqs/2" target="_blank">Graphite</a> 
    * **PR #3** <a href="https://app.graphite.dev/github/pr/mobilecoinofficial/deqs/3" target="_blank">Graphite</a> 
      * **PR #5** <a href="https://app.graphite.dev/github/pr/mobilecoinofficial/deqs/5" target="_blank">Graphite</a> 
        * **PR #6** <a href="https://app.graphite.dev/github/pr/mobilecoinofficial/deqs/6" target="_blank">Graphite</a> 
          * **PR #8** <a href="https://app.graphite.dev/github/pr/mobilecoinofficial/deqs/8" target="_blank">Graphite</a>  👈
          * **PR #7** <a href="https://app.graphite.dev/github/pr/mobilecoinofficial/deqs/7" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/mobilecoinofficial/deqs/8?utm_source=stack-comment).

**Comment by wjuan-mob** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2** <a href="https://app.graphite.dev/github/pr/mobilecoinofficial/deqs/2" target="_blank">Graphite</a> 
    * **PR #3** <a href="https://app.graphite.dev/github/pr/mobilecoinofficial/deqs/3" target="_blank">Graphite</a> 
      * **PR #5** <a href="https://app.graphite.dev/github/pr/mobilecoinofficial/deqs/5" target="_blank">Graphite</a> 
        * **PR #6** <a href="https://app.graphite.dev/github/pr/mobilecoinofficial/deqs/6" target="_blank">Graphite</a> 
          * **PR #8** <a href="https://app.graphite.dev/github/pr/mobilecoinofficial/deqs/8" target="_blank">Graphite</a>  👈
          * **PR #7** <a href="https://app.graphite.dev/github/pr/mobilecoinofficial/deqs/7" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/mobilecoinofficial/deqs/8?utm_source=stack-comment).

File: server/src/bin/main.rs [hunk 1]
```diff
@@ -20,6 +21,12 @@ async fn main() {
 
     let (msg_bus_tx, mut msg_bus_rx) = broadcast::channel::<Msg>(MSG_BUS_QUEUE_SIZE);
     let quote_book = InMemoryQuoteBook::default();
+    // Open the ledger db
```
Comment (eranrund) [inline server/src/bin/main.rs:24]: Nit: I like empty new lines between logical blocks.
```suggestion

    // Open the ledger db
```

**Review by eranrund — APPROVED** [review]

**Review by cbeck88 — APPROVED** [review]

**Comment by cbeck88** [comment]
maybe we should `.gitignore` the `.protoset` file?

**Pull request merged**